client/src/components/visualisation/configMenu/BarConfigMenu.jsx

```diff
@@ -160,6 +161,8 @@ class BarConfigMenu extends Component {
           columnOptions.find(item => item.value === value).title : null,
       }, spec, onChangeSpec, columnOptions)}
     />) : '';
+    const columnsBucketColumn = Immutable.fromJS(filterColumns(columnOptions, ['number', 'text']));
```

Comment:
These are an array of columns, should be plural (columnBucketColumns)?

Reply:
I tried to mean that `columnsBucketColumn ` represents the "columns of bucketColumn"